minio/api.py

```diff
@@ -159,9 +159,6 @@ def __init__(self, endpoint, access_key=None,
             )
         )
 
-    def __del__(self):
```

Comment:
This was added by https://github.com/minio/minio-py/pull/1088. Removal might break

Reply:
Yes on purpose @balamurugana this doesn't build anymore with minio-py master and latest `pylint`